presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -69,7 +70,7 @@ protected Optional<PlanNodeStatsEstimate> doCalculate(TableScanNode node, StatsP
         return Optional.of(PlanNodeStatsEstimate.builder()
                 .setOutputRowCount(tableStatistics.getRowCount().getValue())
                 .setTotalSize(tableStatistics.getTotalSize().getValue())
-                .setConfident(true)
+                .setConfidence(FACT)
```

Comment:
add a TODO here to handle the confidence level properly when filters are pushed into the tablescan.